src/components/FocusTrap/FocusTrapForScreen/index.web.tsx

```diff
@@ -51,18 +40,12 @@ function FocusTrapForScreen({children}: FocusTrapProps) {
                 trapStack: sharedTrapStack,
                 allowOutsideClick: true,
                 fallbackFocus: document.body,
-                // We don't want to ovverride autofocus on these screens.
-                initialFocus: () => {
-                    if (screensWithAutofocus.includes(activeRouteName)) {
-                        return false;
-                    }
-                    return undefined;
-                },
-                setReturnFocus: (element) => {
```

Comment:
We should not return focus to the initial screen if we already have a focused element, e.g. in ReportScreen after closing RHP the Composer should remain focused


https://github.com/Expensify/App/assets/16493223/3f3cb5cd-8c77-49cc-a948-b6675666d024

Reply:
Yes!

Reply:
Works for me but let's also check if `document.activeElement` is `null`. In that case we want to return the focus as well